.claude/agents/operations-specialist.md

```
@@ -998,4 +998,59 @@ As of {{ current_date }}, all systems are operational with the following status:
 - [ ] On-call backup contact established
 - [ ] Emergency escalation procedures reviewed
 - [ ] First maintenance task identified and planned
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)